lua/nordic/groups/integrations.lua

```diff
@@ -348,17 +349,17 @@ function M.get_groups()
     G['@text.note'] = { link = 'Note' }
     --- Markup
     G['@markup'] = { link = '@none' }
-    G['@markup.emphasis'] = { italic = true }
+    G['@markup.emphasis'] = { fg = C.orange.base, italic = true }
```

Comment:
I see why you did this, and I think it makes it easier to edit the rendering (by making those things noticeable). However, I think it should just occur on rendered text because it is distracting otherwise.

Reply:
Fair enough. That might just be a personal preference, but I noticed it adds so much to a markdown if emphasis, bold, and italic are different colors